samples/boards/nrf52/power_mgr/README.rst

```diff
@@ -0,0 +1,58 @@
+.. _power-mgr-sample:
+
+Power management demo
+#####################
+
+Overview
+********
+
+A sample implementation of a power manager app that uses the Zephyr
+power management infrastructure.
+
+This app will cycle through the various power schemes at every call
+to _sys_soc_suspend() hook function.
+It will cycle through the following states:
+
+1. CPU Low Power State
+   A. CONSTANT LATENCY LOW POWER MODE : This Low Power State triggers CONSTLAT
+   task on nrf SOC. In this mode Low Exit Latency. In this Mode 16MHz Clock is
```

Comment:
These lines should be indented to preserve the second-level list you've started.  You'll also need a blank line before the start of the indented list:

     1. CPU Low Power State

        A. CONSTANT LATENCY LOW POWER MODE : This Low Power State triggers CONSTANT
           task on nrf SOC. In this mode Low Exit Latency. ...